day_7/speckled_frogs/10_speckled_frogs.rb

```diff
@@ -0,0 +1,53 @@
+#credit to:
+#user "NoNonsense"
+#October 6, 2014
+#https://stackoverflow.com/questions/19445003/using-ruby-convert-numbers-to-words
+#for number conversion
```

Comment:
@cpfergus1 remember that it's ok to look at other code for inspiration. But we don't want you wholesale copy pasting code you find from the internet.